### DISCUSSION TITLE
Is there a way to group all checked items from Dependency Dashboard into one MR?

### DISCUSSION THREAD

**Discussion by phaninra** [renovatebot/renovate#18326] [Request Help] [answered]
If I understand correctly, the Dependency Dashboard outputs 1 checkbox per 1 potential MR.  So if we applied a `group:all` preset, there'd be only 1 checkbox in the Dependency Dashboard Issue. If we didn't, there'd be many checkboxes, 1 for each potential MR.

Is there a way to configure so that the Dependency Dashboard outputs 1 checkbox per item (as if no grouping was applied), but Renovate will group all checked items into a single MR on a subsequent run?

**Comment by viceice** [comment, marked as answer]
no

**Reply by HonkingGoose** [reply]
We have some open issues that track the idea of grouping updates into one big PR:

- #4404
- #4234
